hmpps-sqs-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -96,14 +97,22 @@ fun isNonStable(version: String): Boolean {
   return isStable.not()
 }
 
-tasks.withType<KotlinCompile> {
-  kotlinOptions {
-    jvmTarget = "16"
+tasks {
+  withType<KotlinCompile> {
+    kotlinOptions {
+      jvmTarget = "11"
+    }
```

Comment:
We could make a future improvement to push this into the root level build gradle file - and maybe for the plugins too - something along these lines https://stackoverflow.com/questions/43810681/inheriting-gradle-plugin-in-subprojects

But needs a bit of playing around and testing, I've not done it before.